lib/views/pages/profile/posts_info/info_card_comment.dart

```diff
@@ -0,0 +1,46 @@
+import "package:flutter/material.dart";
+
+//info card for the comments
+class InfoCardComment extends StatelessWidget {
+  static const cardKey = Key("card");
+
+  const InfoCardComment({
+    super.key,
+    required this.shadow,
+  });
+
+  final BoxShadow shadow;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      key: cardKey,
+      width: 54,
+      height: 80,
+      decoration: BoxDecoration(
+        color: Theme.of(context).colorScheme.secondaryContainer,
+        borderRadius: const BorderRadius.all(Radius.circular(10)),
+        boxShadow: [shadow],
+      ),
+      child: Center(
+        child: ListTile(
+          //TODO: replace with real data
+          title: Text(
+            "Here is a super comment on a super post that talks about something that is super cool and is located in a super spot that is very cosy and nice",
```

Comment:
I think that we can directly add the comment string as input. That way we will be able to directly use widget it as it is :)

```suggestion
  const InfoCardComment({
    super.key,
    required this.shadow,
    required this.comment,
  });

  final BoxShadow shadow;
  final String comment;

  @override
  Widget build(BuildContext context) {
    return Container(
      key: cardKey,
      width: 54,
      height: 80,
      decoration: BoxDecoration(
        color: Theme.of(context).colorScheme.secondaryContainer,
        borderRadius: const BorderRadius.all(Radius.circular(10)),
        boxShadow: [shadow],
      ),
      child: Center(
        child: ListTile(
          title: Text(
            comment,
```